src/pages/components/palette.mdx

```diff
@@ -0,0 +1,59 @@
+---
+title: Palette
+description: >
+  The palette provides the colors used within the Garden design system.
+package: '@zendeskgarden/react-theming'
+components:
+  - theming/src/elements/palette/index.ts
+---
+
+<Usage>
+  <Use>
+    <ul>
+      <li>To reference brand colors specific to Zendesk products</li>
+      <li>
+        <Markdown>To directly access `hue[shade]` color hex values</Markdown>
+      </li>
+    </ul>
+  </Use>
+  <Misuse>
+    <ul>
+      <li>
+        <Markdown>
+          To style custom components with Garden, access the palette through the
+          [`theme.palette`](/components/theme-object#palette) API or
+          [`withTheme`](/components/utilities#withTheme) utility instead
+        </Markdown>
+      </li>
+      <li>
+        <Markdown>
+          To extend Garden with reusable components that respond to theming, use the
```

Comment:
I think you can lose "together":

To extend Garden with reusable components that respond to theming, use the [`getColor`](/components/utilities#getColor) utility ~together~ with the [`theme.colors`](/components/theme-object#colors) API instead